data/korath/korath missions.txt

```diff
@@ -61,3 +61,353 @@ mission "Discovered Korath Space"
 event "label korath space"
 	galaxy "label korath"
 		sprite "label/korath"
+
+mission "Translator Development 1A"
+	name "Translator Development"
+	description "Go to <destination> to ask Eruk if his translation device could be modified for the Wanderers and Korath to use."
+	minor
+	source "Setar Fort"
+	destination "Greenwater"
+	to offer
+		has "Wanderers: Sestor: Final: done"
+	on offer
+		log "People" "Syciil" `Syciil is a Kor Efret ambassador on Setar Fort. She ensures Ektasi Rek has accomodations and whatever else she may need. Along with Rek, she is investigating ways to improve relations between the Wanderers and Kor Efreti.`
+		conversation
+			`When you leave your ship, Rek and a Korath are waiting at the dock. "Hello, Captain <last>. When I was informed you had come to Setar Fort, I came here to see you. This is Scyiil," she says, gesturing to the Korath next to her. Syciil holds up her hands with her palms toward you. "She is an ambassador for the Kor Efreti, and she accompanies me virtually everywhere I go on the planet. I've learned much about the Kor Efreti - and the Korath in general - from her."`
+			choice
+				`	"Pleased to meet you, Syciil. I'm <first> <last>."`
+				`	(Let Rek continue speaking.)`
+					goto nogreeting
+			`	Rek translates the greeting, and Syciil replies. "She says, 'I am pleased to meet you as well. Rek has told me you've seen much of the galaxy.'`
+			label nogreeting
+			`	"You're probably wondering why I was waiting for you. Syciil and I have been discussing how we might improve our peoples' relations. Obviously, our main hindrance is the language barrier. When I was telling her about our recent effort to shut down the Kor Sestor, she asked how you communicate with other Wanderers who did not speak your language. It's strange that this idea never ocurred to me sooner, but I suppose I've been so busy that I haven't had time to think about it. Could your translation device be converted for Wanderer and Korath languages?"`
+			choice
+				`	"To be honest, I'm not sure. A Hai named Eruk gave me it; he might have an answer. That would be a good place to start."`
+				`	"Sorry, I don't know, and I'm not able to help right now. I'll return when I'm available. Good luck."`
+					decline
```

Comment:
There's no reason why the player would want to decline this mission, considering that it has no passenger or cargo or anything requirements.
```suggestion
```